sogs/static/view_room.js

```diff
@@ -51,10 +64,51 @@ const setup = async () => {
                     {
                         throw Error(err);
                     }
-
+                    
                     const plain = Message.decode(data).dataMessage;
+                    // console.log("VERIFY:")
+                    // console.log(data)
+                    // console.log(Message.decode(data))
+                    // console.log(Message.decode(data).dataMessage)
+                    // console.log(Message)
+                    // console.log(Message.verify)
+                    // console.log(Message.verify(data))
+
+                    // reply
+                    if (plain.quote){
+                        originalMsg = document.createElement('p');
+                        originalMsg.classList.add('text-sm', 'italic', 'border-l-2', 'border-sessionGreen', 'pl-2');
+
+                        // originalMsg = document.getElementById('replyTemplate').cloneNode(true);
+
+                        authorId = plain.quote.author
+                        authorId = authorId.substr(authorId.length - 8);
+                        originalMsg.appendChild(document.createTextNode("..." + authorId +": "+plain.quote.text));
+                        e.appendChild(originalMsg);
+                    }
+                    
+                    // message body
                     e.appendChild(document.createTextNode(plain.profile.displayName +": "+plain.body));
+                    e.classList.add('bg-gray-300','dark:bg-lightGray', 'w-fit', 'rounded-lg', 'p-2', 'my-2')
                     elem.appendChild(e);
+                    
+                    // attachments
+                    if (plain.attachments.length > 0){
+                        plain.attachments.forEach(attachment => {
+                            console.log(attachment);
+                            attachmentElement = document.createElement('p');
+                            attachmentElement.appendChild(document.createTextNode("📎\xa0\xa0\xa0"));
```

Comment:
is 0xa0 newline or is that 0x0a ?

Reply:
this is 3 spaces because otherwise the text overlaps with the character - there was probably a better way to do this but i forgot to come back to, and look into it
![Screen Shot 2022-04-14 at 3 32 00 pm](https://user-images.githubusercontent.com/52957382/163320029-df224e4b-e085-4d0c-9516-469cc8e40bfa.png)
t